aten/src/ATen/native/xnnpack/Convolution.cpp

```diff
@@ -208,6 +208,7 @@ ContextConv2D create(
   // NOLINTNEXTLINE(cppcoreguidelines-pro-type-member-init)
   std::array<int64_t, 4> weight_sizes;
 
+  Tensor bias_contig = bias && bias->defined() ? bias->contiguous() : Tensor();
```

Comment:
this would be more efficient using MaybeOwned manually; it could almost be `at::borrow_from_optional_tensor` but that function doesn't do a defined() check